src/utils/todo/action-functions.ts

```diff
@@ -20,6 +20,10 @@ export const doFileNow = (item: TodoItemI) => {
       goToEditUI(path)
       break
     }
+    case FilingTypes.TRANSITION: {
```

Comment:
I think this may be a different Transition filing. Ask Mihai about it.

Reply:
I would expect your filing name to be different:

<img width="418" height="36" alt="image" src="https://github.com/user-attachments/assets/73d6e576-91c2-4b2f-a152-9c1242345cb7" />

Reply:
It is the same thing on the API so it should be the same here as well

Reply:
I leave it with you. I just know that those enum values have been in place since early 2023 😛